test/test.js

```diff
@@ -2,7 +2,7 @@
 var assert = require('assert');
 var fs = require('fs');
 var path = require('path');
-var spawn = require('child_process').spawn;
+var spawn = require('cross-spawn');
```

Comment:
Move this to the end of the list since it isn't a core module (core modules are first).